plugins/cache/network.py

```diff
@@ -0,0 +1,47 @@
+#
+# (c) 2020 Red Hat Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
```

Comment:
Add basic plugin docs

Reply:
On second thoughts, the NetworkCache class is more of a wrapper around the cache plugins defined in ansible/ansible and less of a plugin itself. Maybe we should move it to netcommon utils. If in the future we implement a some network specific cache plugins (a modified version of jsonfile maybe?), then that can be placed under plugins/cache/. What do you think?

Reply:
So it turns out that ansible-test sanity doesn't allow ansible.plugin imports from module_utils code, which means we cannot import cache_loader from ansible-base if we move NetworkCache class to module_utils. Hence, I've moved NetworkCache from cache/network.py to cache/__init__.py.